data/json/npcs/godco/NPC_Chloe_Taylor_King.json

```diff
@@ -0,0 +1,173 @@
+    "responses": [
+      { "text": "I'm sorry for your loss.  What were you saying before?", "topic": "TALK_GODCO_Chloe_Loss" },
+      { "text": "I'm sorry for your loss.  I'd better get going.", "topic": "TALK_GODCO_Chloe_Loss" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Chloe_Loss",
+    "dynamic_line": "They are not dead.",
+    "responses": [ { "text": "…", "topic": "TALK_DONE" } ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Chloe_Mood",
+    "dynamic_line": "I told you already: it sucks.  Nobody's happy here, everyone's hungry, Sonia's songs are getting repetitive, and turns out, the boogeyman is friggin' real.  We can hear it screaming every night.",
+    "responses": [
+      { "text": "What were you saying before?", "topic": "TALK_NONE" },
+      { "text": "I'd better get going.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Chloe_Help",
+    "dynamic_line": "I'm not a baby, you know?  I don't need a babysitter.  Sonia looks out for me though.  She's not so weird, I guess.  She promised to give me guitar lessons and she even signed me an autograph.  I know she's only nice to me because I saved her, but that's okay; we're equal now.",
+    "responses": [
+      {
+        "text": "Saving Sonia sounds like an interesting story; can you tell me what happened?",
+        "topic": "TALK_GODCO_Chloe_Sonia"
+      },
+      { "text": "Aren't you a bit too young to be carrying a knife?", "topic": "TALK_GODCO_Chloe_Knife" },
+      { "text": "What were you saying before?", "topic": "TALK_NONE" },
+      { "text": "I'd better get going.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Chloe_Sonia",
+    "dynamic_line": "I was home alone during the ongoing riots.  My parents chose the worst possible time to reconnect, which sucks, but I'm also kinda happy for them.  Anyway, they were part of the NECC people, so they knew about this place and their plans and all.  Thing is, it was, like, really far away.  I kinda didn't want to go alone but I also didn't want the rioters to eat me, so I asked Sonia - my neighbor - to evacuate with me, I guess.  So that was fun; escaping my hometown, walking down the country road, waiting for a horror to ambush us.  Spoiler alert: we were attacked.  So this zombie shambles in front of us and Sonia whacks it real hard with her guitar!  Its brain splatters all over the pavement, and it was super cool, but I also vomited because it was really disgusting.  We got here soon after that.  My parents weren't here, obviously.  Oh man, they must worry about me so much…",

```

Comment:
```suggestion
    "dynamic_line": "I was home alone during the riots.  My parents chose the worst possible time to reconnect, which sucks, but I'm also kinda happy for them.  Anyway, they were part of the NECC people, so they knew about this place and their plans and all.  Thing is, it was, like, really far away.  I kinda didn't want to go alone but I also didn't want the rioters to eat me, so I asked Sonia - my neighbor - to evacuate with me, I guess.  So that was fun; escaping my hometown, walking down the country road, waiting for a horror to ambush us.  Spoiler alert: we were attacked.  So this zombie shambles in front of us and Sonia whacks it real hard with her guitar!  Its brain splatters all over the pavement, and it was super cool, but I also vomited because it was really disgusting.  We got here soon after that.  My parents weren't here, obviously.  Oh man, they must worry about me so much…",

```